lib/services/ios-project-service.ts

```diff
@@ -186,7 +186,7 @@ export class IOSProjectService extends projectServiceBaseLib.PlatformProjectServ
 	/**
 	 * Exports .xcarchive for AppStore distribution.
 	 */
-	public async exportArchive(projectData: IProjectData, options: { archivePath: string, exportDir?: string, teamID?: string }): Promise<string> {
+	public async exportArchive(projectData: IProjectData, options: { archivePath: string, exportDir?: string, teamID?: string, exportMethod?: string, xcargs?: string }): Promise<string> {
```

Comment:
You've changed the signature of the method, but you don't call it from anywhere with the additional `xcargs` parameter. How do you plan on using the changes?